packages/sanity/src/core/store/_legacy/document/getPairListener.ts

```diff
@@ -77,7 +78,12 @@ export function getPairListener(
   options: PairListenerOptions = {},
 ): Observable<ListenerEvent> {
   const {publishedId, draftId, versionId} = idPair
-
+  if (
+    (idPair.versionId && getVersionFromId(idPair.versionId) === 'published') ||
```

Comment:
I think this will never be true, it implies you could have a `versions.published.<id>` document

Reply:
It actually happened to me yesterday! So leaving it for now – it's just an extra safeguard and easy to remove later on